contributing/README.md

```diff
@@ -104,20 +104,115 @@ Signed-off-by: Random J Developer <random@developer.example.org>
+on the project maintainers and ensures they can be more efficient in their
+reviews.
+
+### Expectation of PR Authors
+
+When your PR is ready for review please notify the assigned reviewers by
+by mentioning them in a comment.
+
+After implementing review feedback, the PR author should notify the reviewer
+by mentioning them in a comment when the PR is ready for another
+review.
+
+If you are not getting a response within a reasonable timeframe, remembering
+that reviewers and maintainers offer their time free of charge and have other
+obligations, you can reach out to the `#crossplane-owners` channel in the
+Crossplane community [Slack] workspace.
+
+### Expectation of Reviewers
+
+If you are assigned as a reviewer on a PR and you are unable to commit to
+reviewing the PR within a reasonable timeframe, you are encouraged to
+communicate this and manage the PR author's expectation.
+
+All reviewers are encouraged to consider the following aspects and to provide
+guidance to the PR author before giving their approval:
+
+* Is the code functionally correct?
+* Is the code written according to the [Coding Style](#coding-style)?
+* Is the solution idiomatically aligned with existing Crossplane APIs?
+* Is the code sufficiently covered by tests?
+* Has the PR author signed the DCO?
+* Are all CI jobs passing?
+
+When providing feedback please consider the following guidelines:
+
+* It helps the recipient a lot to be able to understand the context and
+intention behind comments.
+* Aim to provide feedback in a conversational style, rather than terse
+instructions.
+* Clearly articulate if you are sharing an opinion or instruction that
+needs to be complied with (i.e. a contribution guide rule).
+* Default to asking questions when things are not how we would expect them to
+be. Suggesting rather than demanding changes.
+* Proactively provide context when asking people to change things. Refer to
+where rules are defined or existing precedent exists, where possible.
+* Proactively clarify what needs to change for you to feel comfortable to
+approve a PR - which comments if any are blocking?
+* Allow the reviewee to “win some battles”. Particularly if they’re pushing
+back on something that isn't crucial.
+
+Examples:
+
+* “What do you think about changing X? I think it would be an improvement
+because Y”.
+* "Do we need X at all in this scenario? My thinking is: ..."
+* "I like the direction this is going. I think adding X would be useful
+to Y.""
+* "Please would you make sure your commits are signed (see the DCO check) and
+update the PR description per the template (in particular detail how
+you've tested this change)."
+* "I'm not 100% sure I follow why this is needed - can you add a comment
+(to the code) explaining?"
+* "I might be wrong, but I think what you're actually trying to do here is X"
+
+Being specific with your intention and expectation can save hours of
+undercommunication and misunderstandings.
```

Comment:
This reminds me that another tool available to us is to use a video call to review anything tricky or hard to communicate synchronously. Might be worth mentioning, but not a blocker.

Reply:
I was thinking about adding this, and then more thoughts came up regarding do we mention using Slack calls or just allow folks to self-arrange (likely the best), also what is the threshold for requesting a sync review, and what if reviewees start asking for sync calls too often. In the end I decided to hold off from adding it and feel we can add it in future if we start seeing a pattern emerging that makes sense to encourage.